backend.native/tests/produce_dynamic/kt-36639/main.kt

```diff
@@ -0,0 +1,21 @@
+interface Foo {
+    fun Any.extfoo()
```

Comment:
Is this function accessible from C?
If yes, then this should be checked in the test too.

Reply:
right, added calls to this function too.